scripts/gen_insert_buffer.py

```diff
@@ -81,7 +82,7 @@
             vio_name = drcSections[i].strip()
             report_end_str = re.search("min_report_end", vio_name)
             if report_end_str is not None:
-                print("LLLLLLLLLLLLLLLLLLLLLLLLLLLLLLLLLLLLLLLLLLLLLLLLLLLLLLLLLLLLLL")
+                print("not complete report")
```

Comment:
@donghl17 Can you change this to something like: report is in incomplete

Reply:
I noticed this message when testing. If we hit here, doesn't it mean we collected all the records and it is complete?

Reply:
Longer term, I also wonder if there is a better to parse format we can use, vs the somewhat brittle parsing of the text output.

Reply:
I think this is totally possible. The reports are generated directly out OpenROAD/OpenSTA, so we can have a command that does this without having to parse the reports (which are most likely subject to change). This should be up to the flow owners/maintainers to coordinate.